src/app/(app)/projektek/[id]/page.tsx

```diff
@@ -0,0 +1,30 @@
+import { getPayload } from "payload";
+import config from "@payload-config";
+import CategoryEvents from "@/app/(app)/projektek/Components/category"; 
+
+export default async function CategoryEventsPage({ params }: { params: { id: string } }) {
+    const { id } = params;
```

Comment:
Try it with `const { id } = await params;`

https://nextjs.org/docs/messages/sync-dynamic-apis#possible-ways-to-fix-it